packages/react/runtime/__test__/typecheck/jsx-runtime.test-d.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import type { JSX } from '../../jsx-runtime/index.js';
+import type { MainThread, NodesRef, TouchEvent } from '@lynx-js/types';
+import { assertType, describe, test } from 'vitest';
+import { useMainThreadRef } from '../../lib/lynx-api.js';
```

Comment:
I think it's better to directly use `src/` in tests.

```suggestion
import { useMainThreadRef } from '../../src/lynx-api.js';
```

Reply:
Okay, I'll change it now.